cypress/integration/unit_tests/color_function_util_test.js

```diff
@@ -3,13 +3,14 @@ import {populateColorFunctions, withColor} from '../../../docs/import/color_func
 import {createTrs, setDisasterAndLayers} from '../../support/import_test_util.js';
 import {addFirebaseHooks, loadScriptsBeforeForUnitTests} from '../../support/script_loader.js';
 
-describe('Unit tests for add_disaster page', () => {
+const property = 'color-function';
```

Comment:
nit again :) What I said about declaring as globals for functions doesn't apply to variables: https://google.github.io/styleguide/jsguide.html#features-declared-when-needed. The idea is that functions capture their scopes going up, so you want to make that scope as small as possible (by making them global), while variables are captured by all the scopes going down, so you want to make that as small as possible by making them local.

Reply:
These need to be up here because the functions need to be able to access them. Seems like competing interests!

Reply:
Ah, sorry for being unclear. In my initial nit I was only referring to the "makeTypeDiv" function, since I didn't see it use any variables from the immediate scope, not the other functions you had. The ones that use variables that are mutated in the describe can, of course, live inside the describe!